config.py

```diff
@@ -0,0 +1,25 @@
+# Replace the values in the fields below with your own info

```

Comment:
The function call `load_dotenv(find_dotenv())` loads variables defined in the `.env` file into the current environment. Subsequently, the `os.getenv(<ENV_VAR_NAME>)` call tries to fetch the environment variables. Since they have never been loaded, it will throw an exception. 
Besides, the `os` module has not been imported into this file.